morango/models.py

```diff
@@ -155,24 +155,18 @@ class TransferSession(models.Model):
     """
 
     id = UUIDField(primary_key=True)
-    # partition/filter to know what subset of data is to be synced
-    filter = models.TextField()
-    # is session pushing or pulling data
-    incoming = models.BooleanField()
-    # is this session actively pushing or pulling data?
-    active = models.BooleanField(default=True)
-    chunksize = models.IntegerField(default=500)
```

Comment:
For now we'll use a default. Then we can make that configurable as a Django setting. Then we can have a "pullback" scheme for reducing it in the case of network timeouts etc.

I didn't see any reason to store it, since it's all managed by the client side (whether pushing or pulling), and it maintains its own state in memory.